test/accessibility-test.js

```diff
@@ -19,79 +19,85 @@ describe('app.client.auditAccessibility()', function () {
-          expect(audit.results).to.have.length(0)
-          expect(audit.message).to.equal('Accessibilty audit passed')
-        })
+    it('resolves to an audit object with no results', async function () {
+      await app.client.waitUntilWindowLoaded()
+      const audit = await app.client.auditAccessibility()
+      assert.strictEqual(audit.failed, false)
+      expect(audit.results).to.have.length(0)
+      expect(audit.message).to.equal('Accessibilty audit passed')
     })
   })
 
   describe('when the audit fails', function () {
     beforeEach(function () {
-      return helpers.startApplication({
-        args: [path.join(__dirname, 'fixtures', 'not-accessible')]
-      }).then(function (startedApp) { app = startedApp })
-    })
-
-    it('resolves to an audit object with the results', function () {
-      return app.client.waitUntilWindowLoaded()
-        .auditAccessibility().then(function (audit) {
-          assert.strictEqual(audit.failed, true)
-          expect(audit.results).to.have.length(3)
-
-          expect(audit.results[0].code).to.equal('AX_TEXT_01')
-          expect(audit.results[0].elements).to.deep.equal(['INPUT'])
-          expect(audit.results[0].severity).to.equal('Severe')
-
-          expect(audit.results[1].code).to.equal('AX_HTML_01')
-          expect(audit.results[1].elements).to.deep.equal(['html'])
-          expect(audit.results[1].severity).to.equal('Warning')
-
-          expect(audit.results[2].code).to.equal('AX_COLOR_01')
-          expect(audit.results[2].elements).to.deep.equal(['DIV'])
-          expect(audit.results[2].severity).to.equal('Warning')
+      return helpers
+        .startApplication({
+          args: [path.join(__dirname, 'fixtures', 'not-accessible')]
         })
-        .windowByIndex(1)
-        .auditAccessibility().then(function (audit) {
-          assert.strictEqual(audit.failed, true)
-          expect(audit.results).to.have.length(1)
-
-          expect(audit.results[0].code).to.equal('AX_ARIA_01')
-          expect(audit.results[0].elements).to.deep.equal(['DIV'])
-          expect(audit.results[0].severity).to.equal('Severe')
+        .then(function (startedApp) {
+          app = startedApp
         })
     })
 
-    it('ignores warnings when ignoreWarnings is specified', function () {
-      return app.client.waitUntilWindowLoaded()
-        .auditAccessibility({ ignoreWarnings: true }).then(function (audit) {
-          assert.strictEqual(audit.failed, true)
-          expect(audit.results).to.have.length(1)
-
-          expect(audit.results[0].code).to.equal('AX_TEXT_01')
-          expect(audit.results[0].elements).to.deep.equal(['INPUT'])
-          expect(audit.results[0].severity).to.equal('Severe')
-        })
+    it('resolves to an audit object with the results', async function () {
+      await app.client.waitUntilWindowLoaded()
+      await app.client.windowByIndex(0)
+      var audit = await app.client.auditAccessibility()
```

Comment:
Use `const` instead of `var` here
```suggestion
      const audit = await app.client.auditAccessibility()
```

Reply:
We actually need `const` as we reassign it below!